notebooks/README.md

```diff
@@ -0,0 +1,56 @@
+# Automated checkout using yolov8, text detection and recognition models with OpenVINO™
+
+This Jupyter notebook demonstrates an automated checkout system that integrates object detection and text detection/recognition AI models. Utilizing YOLOv8 for object detection, the notebook efficiently identifies various items within a checkout frame, leveraging its robust and precise detection capabilities. To complement this, the notebook incorporates text detection and recognition within designated regions of interest. This combination allows for the extraction and interpretation of textual information—such as product labels and prices—from the detected items, facilitating a seamless and automated checkout process. The notebook is designed with detailed code explanations and visual outputs, making it accessible for users to understand and adapt the technology for various retail environments.
+
```

Comment:
a pipeline architecture diagram here would be helpful for reader

Reply:
I will add a diagram and gif. Potentially a video later on.